R/pkg/R/SQLContext.R

```diff
@@ -492,6 +492,37 @@ read.text <- function(path, ...) {
   dataFrame(sdf)
 }
 
+#' Create a SparkDataFrame from a csv file.
+#'
+#' Loads a Parquet file, returning the result as a SparkDataFrame.
+#'
+#' @param path Path of file to read. A vector of multiple paths is allowed.
+#' @param ... additional external data source specific named properties.
+#'            You can find the csv-specific options for reading csv files in
+# nolint start
+#'            \url{https://spark.apache.org/docs/latest/sql-data-sources-csv.html#data-source-option}{Data Source Option} in the version you use.
+# nolint end
+#' @return SparkDataFrame
+#' @rdname read.spark.csv
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' path <- "path/to/file.csv"
+#' df <- read.spark.csv(path)
+#' }
+#' @name read.spark.csv
+#' @note read.spark.csv since 3.3.0
+read.spark.csv <- function(path, ...) {
```

Comment:
The problem is that if it's worthwhile adding this different signature given that we're already able to do it with `format`.

Reply:
Yes, We can read csv file by following code:
`df <- read.df("examples/src/main/resources/people.csv", "csv", sep = ";", inferSchema = TRUE, header = TRUE)
`

However, considering that other formats have corresponding advanced functions(read.text() etc.), it is necessary to add a high-level api here.

Reply:
`format ` function looks like just working on scala & java:
https://spark.apache.org/docs/latest/sql-data-sources-load-save-functions.html#manually-specifying-options

Reply:
I have the same feelings as @HyukjinKwon here.

If it wasn't for the name conflict it would be a nice addition. However, adding another naming convention (`spark.read.csv` would be a better choice, but then, it wouldn't be an "obvious" reading utility) just for the sake of omitting `csv` in a call, doesn't seem to be worth it.

Reply:
@zero323  I totally argee. Do we have a more elegant way to support read.csv while maintaining compatibility? If not, I will close this PR.

Reply:
I don't see any clean solution that would follow existing SparkR conventions @deshanxiao.